src/index.ts

```diff
@@ -108,7 +108,7 @@ export default class Combobox {
     const focusIndex = els.indexOf(focusEl)
 
     if ((focusIndex === els.length - 1 && indexDiff === 1) || (focusIndex === 0 && indexDiff === -1)) {
-      this.clearSelection()
+      this.resetSelection()
```

Comment:
Looks like this is the case where the user reaches the end of the options in either direction. Based on current behavior I think we do want to clear the selection in this case, not just reset it:


https://github.com/github/combobox-nav/assets/2294248/0c1f9e77-ce26-4d84-8477-9f19ecb5cb4f

Might be good to add a test for this, but it's not a blocker IMO.



```suggestion
      this.clearSelection()
```

Reply:
If we clear the selection here instead; this will also be a regression in behaviour for the `active` option which previously set the default attribute, in this case, we wouldn't reset that attribute and that may break the option being selected on enter press? 🤔

Reply:
>I think that's an acceptable change - if the first option is active by default but then the user uses the arrow keys to navigate through all the options, it's fine to deselect the first option at that point. That's actually probably preferrable to the current behavior

Reply:
Makes sense! Updated in https://github.com/github/combobox-nav/pull/82/commits/dbf90b53556067ff58182659e549150c2fd9033c#diff-a561630bb56b82342bc66697aee2ad96efddcbc9d150665abd6fb7ecb7c0ab2fL343